django_rq/urls.py

```diff
@@ -5,6 +5,10 @@
 urlpatterns = [
     url(r'^$',
         views.stats, name='rq_home'),
+    url(r'^stats.json/$',
```

Comment:
Can you please combine the two `stats.json` views into one with optional `token` parameter?